crates/ty_ide/src/goto_definition.rs

```diff
@@ -89,6 +178,98 @@ def other_function(): ...
         "#);
     }
 
+    /// goto-definition on a function that's redefined many times in the impl .py
+    ///
+    /// FIXME(?): Currently this yields all instances, presumably it should prefer the last one?
```

Comment:
It might be worth testing how other LSPs (e.g. pylance in VSCode) handle this. I think our handling for this case with both `.py` and `.pyi` should be the same as our handling if there's only the `.py` and no stub-mapping is involved. I believe currently in the latter case we would highlight all definitions, not just the latest one (and that this was an intentional design choice by @UnboundVariable). In general the design choice was that it's confusing for users to highlight different definitions of a symbol depending on location in control flow, and it's better to just highlight all the definitions we can find.

So I think this is probably not a FIXME